### ISSUE TITLE
Bug: secrets do not apply custom Unix permissions

### ISSUE THREAD

**Issue by paulmiro** [pinpox/lollypops#41]
When deploying secrets, the resulting file always has the default `0400` permissions, even when  `lollypops.secrets.files.<name>.mode` is set to another value (for example `0440`).

It also does not throw an error when the option is set to a value that should be invalid

**Comment by pinpox** [comment]
@paulmiro You are right! The change of mode seems to have been lost on some merge, sorry. I added a WIP PR that should fix this, feel free to test and give me some feedback. I'll get around to testing later this week otherwise
